.vscode/extensions.json

```diff
@@ -1,8 +1,8 @@
 {
-    "recommendations": [
-        "streetsidesoftware.code-spell-checker",
-        "dbaeumer.vscode-eslint",
-        "esbenp.prettier-vscode",
-        "xdebug.php-debug"
-    ]
+  "recommendations": [
+    "streetsidesoftware.code-spell-checker",
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode",
+    "xdebug.php-debug"
+  ]
```

Comment:
We should add the formatter here as well for setup

Reply:
done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/6107/commits/bb61e960d79b394cc708513495cb5d549fcafd45

yeah, lots of files but basically all just spacing and commas, which makes for some hideous diffs so I am glad to get that out of the way here

tests and linters all seem happy

Reply:
I had to go through PHP files one-by-one and save, so I didn't get every single one but hopefully the most important ones.

Reply:
Nice, looks like it is working 👍 

![Screenshot 2023-03-29 152036](https://user-images.githubusercontent.com/4127998/228645443-06e1351d-a707-47c4-bce6-fd00003dd7b1.png)